src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php

```diff
@@ -255,7 +255,7 @@ private function createBinaryFileResponse(
     protected function getFileVersion($id, $version)
     {
         /** @var MediaInterface $mediaEntity */
-        $mediaEntity = $this->mediaRepository->findMediaById($id);
+        $mediaEntity = $this->mediaRepository->findMediaByIdForRendering($id, null);
```

Comment:
an issue appears we need to keep here also version in ind as reported in #7319 and #7448